noetic/distribution.yaml

```diff
@@ -8269,6 +8269,18 @@ repositories:
       url: https://github.com/ros-industrial-consortium/tesseract.git
       version: master
     status: developed
+  tf2_2d:
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/locusrobotics/tf2_2d-release.git
+      version: 0.6.2-1
+    source:
+      test_pull_requests: true
```

Comment:
Have you checked http://wiki.ros.org/buildfarm/Pull%20request%20testing?

Reply:
![image](https://user-images.githubusercontent.com/5042191/125484585-8313a614-0943-4545-a1cc-5c905be1d901.png)

![image](https://user-images.githubusercontent.com/5042191/125484706-047389df-d57f-499f-af3e-c38278e38d44.png)

I followed the instructions as I have with other packages for which PR testing is enabled.